src/docker/functions/container/create-container.ts

```diff
@@ -59,6 +60,19 @@ export const createContainer = async (options: CreateContainerOptions): Promise<
 
 type DockerodeEnvironment = string[];
 
+type Volumes = { [volumes in string]: Record<string, unknown> };
+const getVolumes = (volumeNames: string | string[] | undefined) => {
```

Comment:
I think using varargs would simplify this type and its validations (`volumes: ...string`)

Reply:
agreed! done